src/AvaTaxClient.php

```diff
@@ -156,7 +156,7 @@ public function activateAccount($id, $model)
      * 
      * @param int $id The ID of the account to retrieve
      * @param string $include A comma separated list of child objects to return underneath the primary object.
-     * @return AccountModel
+     * @return object
```

Comment:
I think we should be specific with Object types (e.g. AccountModel)  in comment as well as when we specify function parameters. 
It will give clear idea about what Model /Object is expected by function and returned by function to caller. This can also help while auto-generating documents out of comments.